frontend/src/components/InitialAnswerState.js

```diff
@@ -0,0 +1,7 @@
+import {InitialUserState} from "./InitialUserState";
```

Comment:
Use lowercase letters for file names, reserve uppercase names for react components

Reply:
Then these files don't belong in the components folder I guess? Is there any good practice where to put such a file?